src/app/home/home-page.component.html

```diff
@@ -1,7 +1,11 @@
-<div>
+<div class="container-fluid">
   <h1>{{ title }}</h1>
   <hr/>
   <h2>Discover the world and remember every moment. Plan your next trip now!</h2>
+  <br/>
```

Comment:
Instead of br, use css to add a bottom margin to your h2 or a top margin to the rest of the content. We want to avoid html elements where their only purpose is styling. So that means elements like `<strong>`, `<br>`, etc.

Reply:
Just making the comment for when you visit this page. No change necessary.